druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/server/lookup/namespace/cache/MahaNamespaceExtractionCacheManager.java

```diff
@@ -387,6 +390,8 @@ private LookupExtractor getLookupExtractor(final ExtractionNamespace extractionN
             return new JDBCLookupExtractorWithLeaderAndFollower((JDBCExtractionNamespaceWithLeaderAndFollower) extractionNamespace, map, lookupService);
         } else if (extractionNamespace instanceof JDBCExtractionNamespace) {
             return new JDBCLookupExtractor((JDBCExtractionNamespace) extractionNamespace, map, lookupService);
+        } else if (extractionNamespace instanceof RocksDBExtractionNamespace && extractionNamespace.getType() == ExtractionNameSpaceSchemaType.FlatBuffer) {
```

Comment:
@pavanab4u  this is the how it instantiate the RocksDBLookupExtractor supporting flatbuffer schema.

Reply:
I have added ExtractionNameSpaceSchemaType to all existing name spaces in order to state which schema it supports.